package.list

```diff
@@ -22,3 +22,4 @@ usb_cam
 viz
 xsens_driver
 ypspur_ros
+grid_map_ros
```

Comment:
Sorry to be picky but I think the packages should be defined in alphabetical order.

Reply:
Sure, will do.